src/Spryker/Zed/EventBehavior/Business/EventBehaviorFacade.php

```diff
@@ -109,6 +109,26 @@ public function getAvailableResourceNames(): array
         return $this->getFactory()->createEventResourcePluginResolver()->getAvailableResourceNames();
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\EventEntityTransfer[] $eventTransfers
+     * @param string $foreignKeyColumnName
+     *
+     * @return array ['foreignKey' => ['relatedForeignKeyName1' => relatedForeignKeyValue1, ...]]
```

Comment:
Please do not mix examples with descriptions. It's nice to have examples but they need to be in the long-description section. For return type and parameter type descriptions, you should use only describing explanations.

Could you please provide an input + output example? It's a bit complex what happens here. Probably a real example with `product` or something would be better than an abstract example.

Reply:
`getGroupedEventTransferForeignKeysByColumn` receives $eventTransfers and $foreignKeyColumnName, which value will be used for grouping. On the screenshot below, we receive eventTransfers after unassign product from stores. In a group method I group all foreign keys by fk_product_abstract. It is important to keep abstract product to store relation to unassign proper concrete products
https://cl.ly/3e0bede38593